src/coreclr/debug/runtimeinfo/datadescriptor.h

```diff
@@ -105,14 +105,21 @@ CDAC_TYPES_BEGIN()
 
 CDAC_TYPE_BEGIN(Thread)
 CDAC_TYPE_INDETERMINATE(Thread)
+CDAC_TYPE_FIELD(Thread, /*uint32*/, Id, cdac_offsets<Thread>::Id)
```

Comment:
I don't understand this comment type. Is that intentional going forward or a temporary mechanism?